e2e/cypress/integration/files_and_attachments/file_previews_spec.js

```diff
@@ -0,0 +1,493 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @file_and_attachments
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+import {downloadAttachmentAndVerifyItsProperties} from './helpers';
+
+describe('Upload Files', () => {
+    let testTeam;
+
+    beforeEach(() => {
+        // # Login as sysadmin
+        cy.apiAdminLogin();
+
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+            cy.get('.announcement-bar__close').click();
+        });
+    });
+
+    it('JPG', () => {
+        const properties = {
+            route: 'mm_file_testing/Images/JPG.jpg',
+            originalWidth: 400,
+            originalHeight: 479,
+        };
+
+        testImage(properties);
+    });
+
+    it('PNG', () => {
+        const properties = {
+            route: 'mm_file_testing/Images/PNG.png',
+            originalWidth: 400,
+            originalHeight: 479,
+        };
+
+        testImage(properties);
+    });
+
+    it('BMP', () => {
```

Comment:
This fails for me locally. Image is broken.

Reply:
Thanks for confirming. It also fails for me, but I am not sure why. The image is supposed to be fine, and if I upload it manually, it works without any problem. But when I upload it through cypress, the size is different. Same with PSD. Also with PDF, the file is not showing as it should.

Should I remove these tests? Or do you prefer other approach?

Reply:
Yeah, that's weird. My proposal is below where we should split the file into two files.